mesonbuild/mintro.py

```diff
@@ -226,8 +229,13 @@ def run(args):
         testdata = pickle.load(f)
     with open(benchmarkfile, 'rb') as f:
         benchmarkdata = pickle.load(f)
-    with open(installfile, 'rb') as f:
-        installdata = pickle.load(f)
+    # Install data is only available with the Ninja backend
```

Comment:
Installation as a concept doesn't make sense for the Visual Studio backend at least, and I think the same would probably be true for the XCode backend too.

"Installation" into a standardized system prefix is a very Linux/BSD-specific concept.